rest/client_rest.go

```diff
@@ -69,6 +69,10 @@ func (client camundaClientRest) GetAllTasks(processId string) ([]go_camunda_clie
 	}
 	return tasks, err
 }
+func (client camundaClientRest) CompleteTask(taskId string,  variables interface{}) (error) {
+	_, err := client.doRequest("POST", "task/" + taskId + "/complete", variables)
```

Comment:
Is payload really correct?
I dont see where are U serialising payload like (no Marsharl rule, no custom serialization):
```
{
  “variables”: [..]
}
```